homework/index.js

```diff
@@ -27,21 +27,163 @@
         elem.setAttribute(key, value);
       }
     });
+
     return elem;
   }
 
+  function renderRepositoryInfo(repository, leftContainer) {
+    const table = createAndAppend('table', leftContainer);
+    const tbody = createAndAppend('tbody', table);
+    const firstRow = createAndAppend('tr', tbody, {
+      tabindex: '0',
+    });
+    createAndAppend('td', firstRow, {
+      class: 'label',
+      text: 'Repository : ',
+    });
+    const td2 = createAndAppend('td', firstRow);
+    createAndAppend('a', td2, {
+      text: repository.name,
+      href: repository.html_url,
+    });
+
+    const secondRow = createAndAppend('tr', tbody, {
+      tabindex: '0',
+    });
+    createAndAppend('td', secondRow, {
+      class: 'label',
+      text: 'Description :',
+    });
+    createAndAppend('td', secondRow, {
+      text: repository.description,
+    });
+    const thirdRow = createAndAppend('tr', tbody, {
+      tabindex: '0',
+    });
+    createAndAppend('td', thirdRow, {
+      class: 'label',
+      text: 'Forks :',
+    });
+    createAndAppend('td', thirdRow, {
+      text: repository.forks,
+    });
+    const forthRow = createAndAppend('tr', tbody, {
+      tabindex: '0',
+    });
+    createAndAppend('td', forthRow, {
+      class: 'label',
+      text: 'Updated :',
+    });
+    createAndAppend('td', forthRow, {
+      text: repository.updated_at,
+    });
+  }
+
+  function renderRepoContributors(contributors, rightContainer) {
+    createAndAppend('p', rightContainer, {
+      class: 'contributor-header',
+      text: 'Contributors',
+    });
+    contributors.forEach(contributor => {
+      const ul = createAndAppend('ul', rightContainer, {
```

Comment:
You are creating one `ul` per contributor. However, a single `ul` containing the list of contributors would make more sense (also from an ARIA perspective). Then each contributor could be a single `li` and within that `li` you could have whatever elements are needed to render the contributor details.